models/team.js

```diff
@@ -4,7 +4,22 @@ var Schema = mongoose.Schema;
 var teamSchema = Schema({
 	name:String,
 
-	createdDate:Date
+	description:String,
+
+	teamId:String,
+
+	isActive:Boolean,
+
+	project:{
+		type:Schema.Type.ObjectId,
+		ref:'Project'
+	},
+
+	createdDate:Date,
+
+	deactiveDate:Date,
+
+	activeDate:Date
```

Comment:
I was thinking about the create date of the inherited team, but maybe we can use the earliest activeDate as the create date. How do you think @williamyu-xingyunzh ?